.github/workflows/nightly_vtk_master.yml

```diff
@@ -67,58 +67,103 @@ jobs:
         workflow_label: 'producer'
 
 #----------------------------------------------------------------------------
-# MAIN CI: Build and test with a cross-platform build matrix
+# Windows CI: Build and test
 #----------------------------------------------------------------------------
-  ci:
+  windows:
+    needs: [cache_lfs, check_nightly]
+    if: ${{ needs.check_nightly.outputs.should_run == 'true' }}
+    strategy:
+      fail-fast: false
+    runs-on: windows-latest
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        path: 'source'
+        fetch-depth: 0
+        lfs: false
+
+    - name: Generic CI
+      uses: ./source/.github/actions/generic-ci
+      with:
+        vtk_version: ${{needs.check_nightly.outputs.vtk_sha}}
+        raytracing_label: 'raytracing'
+        lfs_sha: ${{ needs.cache_lfs.outputs.lfs_sha}}
+
+#----------------------------------------------------------------------------
+# Linux CI: Build and test
+#----------------------------------------------------------------------------
+  linux:
     needs: [cache_lfs, check_nightly]
     if: ${{ needs.check_nightly.outputs.should_run == 'true' }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu, windows, macos]
         build_type: [standard]
         include:
-          - raytracing_label: raytracing
+          - exclude_deprecated_label: no-exclude-deprecated
           - egl_label: no-egl
-          - os: macos
-            raytracing_label: no-raytracing #598
           - build_type: headless
-            os: ubuntu
-            raytracing_label: raytracing
             exclude_deprecated_label: no-exclude-deprecated
-            bundle_label: no-bundle
             egl_label: egl
+          - build_type: exclude_deprecated
+            exclude_deprecated_label: exclude-deprecated
```

Comment:
do we have to test `exclude-deprecated`? I mean, we can choose a single linux configuration (let's say `commit, egl, no-exclude-deprecated`) and just run this one. As long as all configurations are tested in the regular CI that's fine I guess?

Reply:
> do we have to test exclude-deprecated

Yes we do.

>  I mean, we can choose a single linux configuration 

You mean testing multiple "config flag" in a single CI job ? No, that would be bad design. We should switch "config flag" one by one out of the default for good testing. It would be even better to test the whole matrix but that would be too many builds.

Reply:
Ha, I misunderstood your comment, you actually meant:

> do we have to test exclude-deprecated *in the nightly VTK build*

Let me think about it.

Reply:
You are right, we dont need to test `exclude-deprecated` in the nnightly VTK CI